components/prism-ruby.js

```diff
@@ -102,6 +102,35 @@
 			inside: {
 				'interpolation': interpolation
 			}
+		},
+		{
+			pattern: /<<[-~]?([a-z_]\w*)[\r\n](?:.*[\r\n])*?\s*\1/i,
+			alias: 'heredoc-string',
+			greedy: true,
+			inside: {
+				'delimiter': {
+					pattern: /^<<[-~]?[a-z_]\w*|[a-z_]\w*$/i,
+					alias: 'symbol',
+					inside: {
+						'punctuation': /^<<[-~]?/
+					}
+				},
+				'interpolation': interpolation
+			}
+		},
+		{
+			pattern: /<<[-~]?'([a-z_]\w*)'[\r\n](?:.*[\r\n])*?\s*\1/i,
```

Comment:
Is `<<~'foo'` also allowed?

Reply:
Yeah:

<img width="299" alt="image" src="https://user-images.githubusercontent.com/14028/117237671-c504e500-ae6e-11eb-9d65-006aeb10e865.png">